airflow/provider.yaml.schema.json

```diff
@@ -144,7 +150,7 @@
         "type": "object",
         "properties": {
           "how-to-guide": {
```

Comment:
Is this how-to-guide OK here @mik-laj ? It seems to be at the integration level and it is missing in the operators/sensors/hooks so it seems strange to have it here?

Reply:
Or is it becuse we have source/target integrations here?

Reply:
The transfer operators are treated specially because they related with two integration. The rest can be found in one guide for a specific integration.